### PR TITLE
ServiceRequest: Reflect search by code capability

### PR DESCRIPTION
**Pull request by praneethk2094**
Description
----
* Update the ServiceRequest to reflect the support of search by code.
* Add proprietary code 200 description.
* Fix Nutrition Order and Service Request links to point to R4. 

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by praneethk2094** [comment]
Screen Shot 2020-07-08 at 5 24 14 PM
Screen Shot 2020-07-08 at 5 24 32 PM
Screen Shot 2020-07-08 at 5 26 36 PM
Screen Shot 2020-07-08 at 5 26 48 PM

File: lib/resources/example_json/r4_examples_service_request.rb [hunk 1]
```diff
@@ -42,6 +42,11 @@ module Resources
         "priority": "routine",
         "code": {
           "coding": [{
+            "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/200",
```
Comment (joeratt) [inline lib/resources/example_json/r4_examples_service_request.rb:45]: I think we've got an effort to go through the resources and update the tenant to our new sandbox. not sure if we need to do that here.

Comment (praneethk2094) [reply lib/resources/example_json/r4_examples_service_request.rb:45]: The effort is currently being made through: https://github.com/cerner/fhir.cerner.com/pull/420/.
The assumption is we would like to switch to C1941 in a single go and so no updates are going to be made at this point in time. Thanks!

**Comment by VelocityKnight** [comment]
image

image

image

**Pull request merged**